README.md

```diff
@@ -1,6 +1,36 @@
 # Exasol Test Setup Abstraction
 
-Common interface for testing with an Exasol database in local and AWS environments using Java.
+[![Build Status](https://travis-ci.com/exasol/exasol-test-setup-abstraction.svg?branch=main)](https://travis-ci.com/exasol/exasol-test-setup-abstraction)
+
+[![Quality Gate Status](https://sonarcloud.io/api/project_badges/measure?project=com.exasol%3Aexasol-test-setup-abstraction&metric=alert_status)](https://sonarcloud.io/dashboard?id=com.exasol%3Aexasol-test-setup-abstraction)
+
+This repository defines a standardized interface for accessing an Exasol database for testing.
+
+That allows you to write tests that allow you to switch the test-backend.
```

Comment:
```suggestion
That allows you to write tests where you can to switch the test-backend.
```
Maybe we can explain a bit what is the test-backend, as it might not be fully clear for someone new to the project

Reply:
```suggestion
That allows you to write tests where you can to switch the test-backend. For you can weitch between an Exasol cluster in the cloud or a local docker-db.
```